.pylint.rc

```diff
@@ -0,0 +1,58 @@
+[MASTER]
+jobs=4
+persistent=yes
+suggestion-mode=yes
+unsafe-load-any-extension=no
+
+# Blacklist files or directories (basenames, not paths)
+ignore=
+
+# blacklist files or directories by regex  (basenames, not paths)
+ignore-patterns=
+
+[EXCEPTIONS]
+
+overgeneral-exceptions=Exception,RaidenUnrecoverableError
+
+[BASIC]
+
+bad-names=foo,bar,baz,toto,tutu,tata
+good-names=i,j,k,_
+
+[LOGGING]
+
+logging-modules=logging,structlog
+
+[MESSAGES CONTROL]
+
+disable=all
+enable=no-value-for-parameter,too-many-format-args,no-member
+
+[REPORTS]
+
+reports=no
+score=no
+
+[FORMAT]
+
+expected-line-ending-format=LF
+ignore-long-lines=^\s*(# )?<?https?://\S+>?$
+max-line-length=100
+max-module-lines=1000
+no-space-check=trailing-comma
```

Comment:
removed the default `dict-separator` which allows for tabular dictionaries:

```python
d = {1  : 'a',  # <- this would be valid
     222: 'b'}
```